zeroex/ordervalidator/order_validator_bench_test.go

```diff
@@ -0,0 +1,74 @@
+	order := zeroex.Order{
+		ChainID:               big.NewInt(constants.TestChainID),
+		MakerAddress:          constants.GanacheAccount1,
+		TakerAddress:          constants.NullAddress,
+		SenderAddress:         constants.NullAddress,
+		FeeRecipientAddress:   constants.NullAddress,
+		MakerAssetData:        multiAssetAssetData,
+		MakerFeeAssetData:     multiAssetAssetData,
+		TakerAssetData:        multiAssetAssetData,
+		TakerFeeAssetData:     multiAssetAssetData,
+		Salt:                  big.NewInt(int64(time.Now().Nanosecond())),
+		MakerFee:              big.NewInt(0),
+		TakerFee:              big.NewInt(0),
+		MakerAssetAmount:      big.NewInt(100),
+		TakerAssetAmount:      big.NewInt(42),
+		ExpirationTimeSeconds: big.NewInt(time.Now().Add(24 * time.Hour).Unix()),
+		ExchangeAddress:       ethereum.GanacheAddresses.Exchange,
+	}
+	signedOrder, err := zeroex.SignTestOrder(&order)
```

Comment:
Optional Nit: You can probably use the `scenario` package to simplify this.

Reply:
It doesn't work for `*testing.B`? Also, this is in the init, so it doesn't have access to `b` yet?

Reply:
Good point. Forgot about that.